metabase/metabase
# Query validator FE

Closes https://github.com/metabase/metabase/issues/45546

Depends on https://github.com/metabase/harbormaster/issues/5156

### Description
Adds a UI for the Query Field Validation Feature. This will display queries with broken data references in a table under `Admin -> Troubleshooting -> Query Validation`. 

### How to verify
The easiest way to create an invalid card is to do so via SQL Queries. Go to New -> SQL Query and type `Select foo from Orders`. Save the card, then go to  `Admin -> Troubleshooting -> Query Validation`.  Your card should be present.

### Demo
![image](https://github.com/user-attachments/assets/c9118822-1d4f-4a44-bec0-4b3bc961d426)
(pretend creator is populated above)

### Checklist
